### DISCUSSION TITLE
MSI - how to prevent creation of desktop and start menu shortcuts?

### DISCUSSION THREAD

**Discussion by cloudhal** [rustdesk/rustdesk#8829] [General]
Deploying remotely, I don't want to clutter the PC with shortcuts.

**Comment by fufesou** [comment]
I'll add options, both UI and command line.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/pull/8949
https://github.com/rustdesk/rustdesk/pull/9001

**Reply by cloudhal** [reply]
Thank you for adding this!
